etl/steps/data/explorers/migration/2024-08-05/migration_config_dict.py

```diff
@@ -0,0 +1,866 @@
+P_TOTAL = "Total number"
```

Comment:
I totally understand why you would want to have all this as a separate file. However, there is a downside: If you make changes in this auxiliary file, ETL will not notice them and will not re-execute the step. So, to avoid that, it would make more sense to put all this information in the same step. But if you prefer to keep it this way, that's fine, I'm just pointing out that caveat (at least that's how it used to be, maybe ETL has improved to be able to tackle these cases, have you tried?).

Reply:
Yes, that makes sense! I don't think it re-executes the step automatically (at least not in my experience) but as this file should only change if something in the upstream (data) steps changes, I think it's fine to leave it in a separate file for now. Thanks for the heads-up though!